### PR TITLE
Kill tool dependencies

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
ping @mmiladi

**Comment by mmiladi** [comment]
@bgruening is tool_dependencies a pre-Conda era?

**Comment by bgruening** [comment]
Yes. It clears a lot in your admin panel.

File: tools/rna_tools/cofold/tool_dependencies.xml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0"?>
-<tool_dependency>
-    <package name="cofold" version="0.0.1">
```
[the remark this answers is not in the document]
Comment (bgruening) [reply tools/rna_tools/cofold/tool_dependencies.xml]: Cofold was updated long time ago, see here: https://github.com/bgruening/galaxytools/blob/master/tools/rna_tools/cofold/cofold.xml#L4

[the remark this answers is not in the document]
Comment (mmiladi) [reply tools/rna_tools/cofold/tool_dependencies.xml]: Hmm. Ok so these dependency files are really zombies! :D

**Comment by mmiladi** [comment]
Is this also probably related?
https://github.com/bgruening/galaxytools/blob/master/packages/package_rnashapes_3_2_5/tool_dependencies.xml

**Comment by bgruening** [comment]
If packages are that common that the installation instruction could be shared between different tools we create so called `packages` ... than a tool could depend on a `package`... so related yes, but the entire `packages` folder is deprecated. We will not remove support but a new tool should not depend anymore on it.

**Comment by mmiladi** [comment]
I see. Thanks much for clarification and the work. Looks good   :)

**Pull request merged**